.rubocop.yml

```diff
@@ -300,3 +300,10 @@ Style/WhenThen:
 Style/ZeroLengthPredicate:
   Description: 'Use #empty? when testing for objects of length 0.'
   Enabled: true
+
+
+Lint/RequireParentheses:
+  Description: >-
+                 Use parentheses in the method call to avoid confusion
+                 about precedence.
+  Enabled: true
```

Comment:
Did you mean to check this in? It's unrelated, and we wouldn't add this anyways since it doesn't differ from the default Rubocop config.